libs/langchain/tests/unit_tests/chat_models/test_base.py

```diff
@@ -1,7 +1,16 @@
+import os
+
 import pytest
+from langchain_core.language_models import BaseChatModel
+from langchain_core.messages import HumanMessage
+from langchain_core.runnables import RunnableConfig
 
 from langchain.chat_models.base import __all__, init_chat_model
 
+os.environ["OPENAI_API_KEY"] = os.environ.get("OPENAI_API_KEY", "dummy")
```

Comment:
test code should never set env variables (especially unit test code), could you use mock or a context manager?